platform/platform.go

```diff
@@ -1,21 +1,34 @@
 package platform
 
 import (
+	"fmt"
+
 	"github.com/buildpacks/lifecycle/api"
-	"github.com/buildpacks/lifecycle/cmd"
-	v05 "github.com/buildpacks/lifecycle/platform/v05"
+	"github.com/buildpacks/lifecycle/platform/common"
+	pre06 "github.com/buildpacks/lifecycle/platform/pre06"
```

Comment:
```suggestion
	"github.com/buildpacks/lifecycle/platform/pre06"
```
(redundant alias)

Reply:
I'm not sure why the linter doesn't also think the aliases for v06 and v07 are redundant...